src/include/cephfs/libcephfs.h

```diff
@@ -107,6 +107,8 @@ struct ceph_mount_info;
 struct ceph_dir_result;
 struct CephContext;
 
+typedef int32_t fs_cluster_id_t;
```

Comment:
Let's go ahead and define this as an `int64_t` just so it's future proof if we change the Filesystem::fscid to 64bits later.